quiz.py

```diff
@@ -1,6 +1,56 @@
 """
 File: quiz.py
-Author: Name
-Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Author: Chiara A
+Date: 2024/02/01
+Description: This program is a multiple quiz game!
+"""
+
+print("Multiple choice quiz game!\n")
+
+
+user_score = 0
+total_questions = 4
+questions_correct = 0
+
+question_one = input("What is my nick name?: \na) Chiara \nb) kiki \nc) chichi \nd) ki \n")
```

Comment:
Clever use of `\n` special character to do line breaks in a single statement. It's efficient, but can make the code a little. You can use the `\` special character to break a line across multiple lines, like this:

```python
question_one = input("What is my nick name?: \
                     \na) Chiara \
                     \nb) kiki \
                     \nc) chichi \
                     \nd) ki \n")
```

Note the indentation to make the printed output easier to read in a code review.